server/src/internalClusterTest/java/org/opensearch/index/mapper/StarTreeMapperIT.java

```diff
@@ -356,10 +363,19 @@ private XContentBuilder getMappingWithDuplicateFields(boolean isDuplicateDim, bo
     }
 
     private static String getDim(boolean hasDocValues, boolean isKeyword) {
+        if (hasDocValues) {
+            return random().nextBoolean() ? "numeric" : "keyword";
+        } else if (isKeyword) {
+            return "ip";
+        }
+        return "numeric_dv";
+    }
+
+    private static String getMetric(boolean hasDocValues, boolean isKeyword) {
         if (hasDocValues) {
             return "numeric";
         } else if (isKeyword) {
-            return "keyword";
+            return "ip";
```

Comment:
why are we returning "ip" in the above two ifs?

Reply:
now that keyword is supported, ip is something i'm returning as an 'unsupported' dimension. once ip gets supported next, will change to a different mapping type.